scripts/repl.js

```diff
@@ -77,19 +77,21 @@
   /*
    * Long term storage for persistence of state/etc
    */
-  function StorageService () {
-    this.store = window.localStorage;
+  function StorageService () {}
+
+  StorageService.prototype.getStore = function () {
+    return window.localStorage;
   }
 
   StorageService.prototype.get = function (key) {
     try {
-      return JSON.parse(this.store.getItem(key));
+      return JSON.parse(this.getStore().getItem(key));
     } catch(e) {}
```

Comment:
If think we should emit a warning here. It might be useful for debugging purposes.